core/src/main/java/com/devonfw/tools/solicitor/componentinfo/scancode/UncuratedScancodeComponentInfoProvider.java

```diff
@@ -141,100 +147,120 @@ private ScancodeComponentInfo parseAndMapScancodeJson(String packageUrl, Scancod
+	      int numberOfGplLicenses = 0;
+	      for (JsonNode li : file.get("licenses")) {
+	        String licenseName = li.get("spdx_license_key").asText();
+	        if ("Classpath-exception-2.0".equals(licenseName)) {
+	          classPathExceptionExists = true;
+	        }
+	        if (!spdxIdMap.containsKey(licenseName)) {
+	          spdxIdMap.put(licenseName, licenseName);
+	          if (licenseName.startsWith("GPL")) {
+	            numberOfGplLicenses++;
+	          }
+	        }
+	      }
+	      if (classPathExceptionExists) {
+	        if (numberOfGplLicenses == 0) {
+	          LOG.warn(LogMessages.CLASSPATHEXCEPTION_WITHOUT_GPL.msg(), packageUrl);
+	        } else if (numberOfGplLicenses > 1) {
+	          LOG.warn(LogMessages.CLASSPATHEXCEPTION_MULTIPLE_GPL.msg(), packageUrl);
+	        } else {
+	          LOG.debug("Adjusting GPL license to contain WITH Classpath-execption-2.0 for " + packageUrl);
+	          for (String licenseName : spdxIdMap.keySet()) {
+	            if (licenseName.startsWith("GPL")) {
+	              spdxIdMap.put(licenseName, licenseName + " WITH Classpath-exception-2.0");
+	            }
+	          }
+	          // do not output the Classpath-exception-2.0 as separate License
+	          spdxIdMap.remove("Classpath-exception-2.0");
+	        }
+	      }
+	      for (JsonNode li : file.get("licenses")) {
+	        String licenseid = li.get("key").asText();
+	        String licenseName = li.get("spdx_license_key").asText();
+	        String effectiveLicenseName = spdxIdMap.get(licenseName);
+	        if (effectiveLicenseName == null) {
+	          // not contained in map --> this must be the Classpath-exception-2.0
+	          continue;
+	        } else {
+	          licenseName = effectiveLicenseName;
+	          if (licenseName.endsWith("WITH Classpath-exception-2.0")) {
+	            licenseid = licenseid + "WITH Classpath-exception-2.0";
+	          }
+	        }
+	        String licenseDefaultUrl = li.get("scancode_text_url").asText();
+	        licenseDefaultUrl = normalizeLicenseUrl(packageUrl, licenseDefaultUrl);
+	        double score = li.get("score").asDouble();
+	        String licenseUrl = file.get("path").asText();
+	        int startLine = li.get("start_line").asInt();
+	        int endLine = li.get("end_line").asInt();
+	        if (!takeCompleteFile) {
+	          licenseUrl += "#L" + startLine;
+	          if (endLine != startLine) {
+	            licenseUrl += "-L" + endLine;
+	          }
+	        }
+	
+	        licenseUrl = normalizeLicenseUrl(packageUrl, licenseUrl);
+	        String givenLicenseText = null;
+	        if (licenseUrl != null) {
+	          givenLicenseText = this.fileScancodeRawComponentInfoProvider.retrieveContent(packageUrl, licenseUrl);
+	        }
+	
+	        componentScancodeInfos.addLicense(licenseid, licenseName, licenseDefaultUrl, score, licenseUrl,
+	            givenLicenseText, endLine - startLine);
+	      }
+      
```

Comment:
Your editor introduced tabs in addition to spaces. This disturbes formatting/indentation and shows these lines as changed. Please change you editor setting so that formatting is preserved. Only lines which really are changes should be shown as changed. Use Spaces instead of Tabs. (Please check for all modified files).